openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -0,0 +1,169 @@
+import unittest
+import ddt
+
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.conf import settings
+from rest_framework.test import APITestCase, APIClient
+
+from student.tests.factories import UserFactory
+from student.models import UserProfile
+
+TEST_PASSWORD = "test"
+
+@ddt.ddt
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
```

Comment:
Why is this test only valid in the LMS? We are exposing this API in Studio too so we should really be testing it in both.